artiq/frontend/artiq_coremgmt.py

```diff
@@ -73,6 +73,10 @@ def get_argparser():
                          help="key and file whose content to be written to "
                               "core device config")
 
+    subparsers.add_parser("write_ch_names",
```

Comment:
What's the workflow when one uses ``artiq_mkfs``?
This code might have to be broken out into a separate frontend program.

Reply:
Maybe I put the string processing code in `coremgmt` into a function for `mkfs` to call?

Reply:
No

Reply:
For people using `artiq_mkfs`, now can add argument `-e write-ch-names` to do the same thing.

Reply:
Just make a separate program.